app/forms/search_form.rb

```diff
@@ -4,10 +4,10 @@ class SearchForm
   include ActiveModel::Validations
 
   validates :q, presence: true
-  validates :area_of_law, presence: true, if: :postcode_search?
+  validates :area_of_law, presence: true, if: :postcode_search
```

Comment:
from the [code](https://github.com/ministryofjustice/courtfinder/blob/old_prod_working/lib/court_search.rb#L81) I understand that the method is just checking if it is a postcode search. So, maybe `is_postcode_search?` ?